mappings/net/minecraft/client/gui/screen/Screen.mapping

```diff
@@ -71,11 +74,18 @@ CLASS net/minecraft/class_437 net/minecraft/client/gui/screen/Screen
 		ARG 1 message
 		ARG 2 toHud
 	METHOD method_25426 init ()V
+		COMMENT Called when a screen should be initialized.
+		COMMENT
+		COMMENT <p>This method is called when this screen is {@link MinecraftClient#openScreen(Screen) opened} or resized.
 	METHOD method_25427 sendMessage (Ljava/lang/String;)V
 		ARG 1 message
 	METHOD method_25428 confirmLink (Z)V
 		ARG 1 open
 	METHOD method_25429 addChild (Lnet/minecraft/class_364;)Lnet/minecraft/class_364;
+		COMMENT Adds a child element to this screen.
+		COMMENT If the child element is an {@link net.minecraft.client.gui.widget.AbstractButtonWidget}, you should use {@link Screen#addButton(AbstractButtonWidget)} instead.
+		COMMENT
+		COMMENT <p>With the exception of button-like widgets, adding a child element to a screen does not garuntee the widget is rendered or ticked.
```

Comment:
it should be that using this method will absolutely not be rendered automatically, addButton adds the button to the buttons list that does it